### PR TITLE
Fixing the sorting of database files

### PR DESCRIPTION
**Pull request by litghost**
Copy of #1226, remove use of xjson from `074`
>  - Rework how the `json` files are sorted (numbers are treated as numerics).
>  - Sort `csv` and `txt` files.
>  - Sort `segbits.*origin_info.db` files.
>  - Sort the grid file.
> 
> How this changes the output can be seen in https://github.com/SymbiFlow/prjxray-db/pull/11/files

### CODE REVIEW

**Comment by mithro** [comment]
@litghost Yeah - I was thinking that xjson might be the issue on that script.

File: fuzzers/074-dump_all/create_node_tree.py [hunk 1]
```diff
@@ -273,7 +272,7 @@ def main():
 
     print('{} Writing node tree'.format(datetime.datetime.now()))
     with open(os.path.join(args.output_dir, 'node_tree.json'), 'w') as f:
-        xjson.pprint(f, nodes)
+        json.dump(nodes, f, indent=2, sort_keys=True)
```
Comment (mithro) [inline fuzzers/074-dump_all/create_node_tree.py]: Since we are going to sort + indent things at a later date, should we just skip indent / sort_keys?

Comment (litghost) [reply fuzzers/074-dump_all/create_node_tree.py]: It's still convenient for debugging to have the `indent=2`, and the sort_keys should also be fast.

**Comment by litghost** [comment]
@mithro looks like we are in business

**Comment by mithro** [comment]
I have pushed an updated database based on this change at https://github.com/SymbiFlow/prjxray-db/commit/db494e9d5d71edd3c8b6558f7eec12e163aef2d8

**Pull request merged**